src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -122,6 +122,7 @@ class SidebarScreen extends Component {
     }
 
     render() {
+        const workspaces = _.filter(this.props.allPolicies, policy => policy && policy.type === CONST.POLICY.TYPE.FREE);
```

Comment:
I see this logic is used in multiple places. it is not clear why we are filtering the `CONST.POLICY.TYPE.FREE` policies unless we have a comment. 

So  better would be create a action method in action/Policy.js `getWorkspacePolicies(policies: Array<Policy>): Array<WorkspacePolicy>`. Which does the same thing but you leave a comment the same as on the New Room Page and then replace all the places where we are getting workspaces from policies with this method call.

Reply:
I did think about moving it to action but wasn't sure. 

https://github.com/Expensify/App/blob/d9355db0e9685021c3be1debca3c43751ad2d759/src/pages/workspace/WorkspaceNewRoomPage.js#L61-L65

`WorkspaceNewRoomPage` is where we have the same code, and a small comment which says `Workspaces are policies with type === 'free'`.

What happens is, even for a new user a policy exist of `type === personal`, with a default name like "Manan jadhav's expenses"

Reply:
Maybe just leave it for now and add a small comment. That should be enough. There are other places where we check the free policy such as InitialSettingsPage. But adding another filter call would be extra thing to do as we can filter the list based on multiple things. 

So Just a comment should be enough..

Reply:
I've added the comment